python/lsst/daf/butler/registry/_registry.py

```diff
@@ -657,84 +657,86 @@ def find(self, collection: str, datasetType: Union[DatasetType, str], dataId: Op
         return self._makeDatasetRefFromRow(result, datasetType=datasetType, dataId=dataId)
 
     @transactional
-    def addDataset(self, datasetType: Union[DatasetType, str],
-                   dataId: DataId, run: str, producer: Optional[Quantum] = None,
-                   recursive: bool = False, **kwds: Any) -> DatasetRef:
-        """Adds a Dataset entry to the `Registry`
+    def insertDatasets(self, datasetType: Union[DatasetType, str], dataIds: Iterable[DataId],
+                       run: str, *, producer: Optional[Quantum] = None, recursive: bool = False
+                       ) -> List[DatasetRef]:
```

Comment:
github seems to be showing a darker green rectangle, is it a tab?

Reply:
I think those are just new spaces.